prover/config.yml

```diff
@@ -17,3 +17,5 @@ metrics_port: 9100
 enable_dangerous_logging: false
 enable_debug_checks: false
 enable_federated_jwks: false
+disable_exp_claim_in_future_check: false
+disable_iat_in_past_check: false
```

Comment:
For the same reasons, rename this to `enable_jwt_iat_validation`.

Reply:
Again, if you feel it is important to define what "JWT `iat` validation" does, rename to `enable_jwt_iat_not_in_future_check`